ros_alarms/alarms.py

```diff
@@ -117,10 +118,12 @@ def get_alarm(self):
 
     def _severity_cb_check(self, severity):
```

Comment:
Checking once at the beginning of the function would be cleaner and return False if there was no last alarm.
That being said I don't think we should ever set up a case where the last alarm is none. Resetting last alarm in the `get_alarm` function would be a good addition as well.